docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md

```diff
@@ -0,0 +1,29 @@
+---
+aliases:
+  - /docs/grafana/latest/dashboards/build-dashboards/modify-dashboard-settings/
+title: Modify dashboard settings
+menuTitle: Dashboard settings
+weight: 8
+keywords:
+  - xxx
+---
+
+# Modify dashboard settings
+
+xxxxx.
```

Comment:
@chri2547 Missing content, presumably also missing on the main branch.

Reply:
> @chri2547 Missing content, presumably also missing on the main branch.

Oh thank you for this catch. I'll need to update that in a separate PR after this one is merged.